src/arkit.cs

```diff
@@ -576,7 +576,9 @@ interface ARSession {
 		[NullAllowed, Export ("configuration", ArgumentSemantic.Copy)]
 		ARConfiguration Configuration { get; }
 
-		// 'runWithConfiguration:' selector marked as unavailable in Xcode 9 beta 5. Use 'Run (ARConfiguration configuration, ARSessionRunOptions options)' instead.
+		[Export ("runWithConfiguration:")]
+		void Run (ARConfiguration configuration);
```

Comment:
That's fine since `NS_SWIFT_UNAVAILABLE` is not the same as being unavailable (like in ObjC)